### PR TITLE
Remove grammar support for CREATE/DROP DATABASE

### PR DESCRIPTION
**Pull request by stephaniewang526**
DuckDB does not need these grammar. Removing them allows the syntax to fail through to the extensions to be handled.

### CODE REVIEW

File: scripts/generate_grammar.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def concat_dir(dname, extension, add_line_numbers=False):
 cmd += ["-o", result_source, "-d", target_file]
 print(' '.join(cmd))
 proc = subprocess.Popen(cmd, stderr=subprocess.PIPE)
-res = proc.wait()
+res = proc.wait(timeout=10)
```
Comment (rjatwal) [inline scripts/generate_grammar.py:252]: Be good to add a reason for the timeout i.e. to ensure CI does not hang as was seen when running with Bison 3.x release.

**Review by rjatwal — APPROVED** [review]
LGTM

File: test/sql/create/create_database.test [hunk 2]
```diff
@@ -7,9 +7,14 @@ require noforcestorage
 statement error
 CREATE DATABASE mydb;
 ----
-CREATE DATABASE not supported in DuckDB yet
```
Comment (stephaniewang526) [inline test/sql/create/create_database.test:10]: I updated the test cases here to ensure that these syntax will fail through to the extension but I can also see you may not want these tests anymore so I'm happy to remove this file too. LMK.

File: scripts/generate_grammar.py (hunk 1 above)
Comment (stephaniewang526) [reply scripts/generate_grammar.py]: done!

**Review by carlopi — CHANGES_REQUESTED** [review]
Hi!
You will have to commit changes also to`src/parser/transform/statement/CMakeLists.txt` to avoid cmake errors, like:
```
diff --git a/src/parser/transform/statement/CMakeLists.txt b/src/parser/transform/statement/CMakeLists.txt
index f6760fdf85..eba9c43e9a 100644
--- a/src/parser/transform/statement/CMakeLists.txt
+++ b/src/parser/transform/statement/CMakeLists.txt
@@ -14,7 +14,6 @@ add_library_unity(
   transform_create_table.cpp
   transform_create_view.cpp
   transform_create_type.cpp
-  transform_create_database.cpp
   transform_delete.cpp
   transform_explain.cpp
   transform_export.cpp
```
And potentially also from `./github/config/uncovered_lines.csv` could be cleaner (but unsure if this second is actually required).

**Comment by stephaniewang526** [comment]
> Hi! You will to commit changes also to`src/parser/transform/statement/CMakeLists.txt` to avoid cmake errors, like:
> 
> ```
> diff --git a/src/parser/transform/statement/CMakeLists.txt b/src/parser/transform/statement/CMakeLists.txt
> index f6760fdf85..eba9c43e9a 100644
> --- a/src/parser/transform/statement/CMakeLists.txt
> +++ b/src/parser/transform/statement/CMakeLists.txt
> @@ -14,7 +14,6 @@ add_library_unity(
>    transform_create_table.cpp
>    transform_create_view.cpp
>    transform_create_type.cpp
> -  transform_create_database.cpp
>    transform_delete.cpp
>    transform_explain.cpp
>    transform_export.cpp
> ```
> 
> And potentially also from `./github/config/uncovered_lines.csv` could be cleaner (but unsure if this second is actually required).

Done. Thanks for cathcing it. Not sure how I was able to compile... PTAL. Thanks!

**Review by carlopi — APPROVED** [review]
I think there are two sides here: approach and implementation.

Approach I miss some context, so hard to say anything. It seems to me more general to have special handling in duckdb-core, and I am not sure I get what was not workable of the previous situation.

Implementation side this makes sense, cutting lines / complexity is always nice. And I believe keeping the tests in also makes sense.

If the approach has sense, for me is a green light.

**Comment by stephaniewang526** [comment]
> I think there are two sides here: approach and implementation.
> 
> Approach I miss some context, so hard to say anything. It seems to me more general to have special handling in duckdb-core, and I am not sure I get what was not workable of the previous situation.
> 
> Implementation side this makes sense, cutting lines / complexity is always nice. And I believe keeping the tests in also makes sense.
> 
> If the approach has sense, for me is a green light.

Thanks for the review. I had an offline conversation with @Mytherin on the approach. I think in general it will be easier to just handle everything in the extension instead of having to come to DDB to make changes for the transformer and bind logic.

**Comment by carlopi** [comment]
Thanks for the context / update. This I'd say this is good to go!

**Comment by Mytherin** [comment]
Thanks! LGTM

**Pull request merged**